### ISSUE TITLE
"Last File Received" shows wrong info

### ISSUE THREAD

**Issue by uok** [syncthing/syncthing#1952]
"Last File Received" (sometimes?) shows the wrong info (only after deletion?)

- device A shares file1.txt with device B
- device B shares file2.txt with device A
- sync
- device B deletes file1.txt
- sync
- on device A "Last File Received" shows: file1.txt --- should be: file2.txt

**Comment by ralder** [comment]
today also see it :) and think about change  "Last File Received" to "Last File Handled"

**Comment by calmh** [comment]
"Last Change Received" maybe

**Comment by uok** [comment]
I don't want to change the label - but show correct information at "last file received"
e.g. on device A
:white_check_mark: if a new file is received from device B
:white_check_mark:  if a change is received from device B
:x: if a file is deleted on device B

and as suggested in #1883 a separate area with "last file sent"

**Comment by calmh** [comment]
Yeah... That action-icon + name looks nice.

**Comment by uok** [comment]
Let me try again: if file1.txt is deleted on device B, it shows as received on device A which is a bug


![lastreceived](https://cloud.githubusercontent.com/assets/9367272/8179555/b8d1f932-1417-11e5-8267-29badb6853da.png)

**Comment by Eddy2909** [comment]
its not a bug, its a wrong wirding of the gui

**Comment by AudriusButkevicius** [comment]
Yeah, I guess we should just change the wording here, or add an icon, as you are suggesting.

**Comment by uok** [comment]
ok, I understand ... IMHO it is better to see only what's new
how many changes back are saved? maybe it's better to have a little modal onclick that shows the last 10 (?) new/changed/deleted files and rename the label to "last changes"
